packages/ng/src/app/date-range-picker/date-range-modal/date-range-modal.component.html

```diff
@@ -1,7 +1,6 @@
 <div class="date-range-modal-inputs">
 	<div class="input mod-inline">
 		<input class="input-field"
-					 [ngClass]="{'is-filled': !!start}"
```

Comment:
@sanlucca This is used to keep label on top position if the input is filled, and switch to error style if it's empty, correct ? In this case we'd like the former to happen, but not the latter.

Reply:
The "is-error" state and the "is-filled" state aren't correlated. It can be both filled and error.